### DISCUSSION TITLE
Custom marker placement

### DISCUSSION THREAD

**Discussion by cgombola** [Leaflet/Leaflet#9464] [General]
I am a complete amateur, I have maps and basic markers working wonderfully.  I am trying to place a custom marker on a map with coordinates, which works, but, that custom marker stays in its original place on the screen when I zoom in and our, rather than staying on it's coordinate.  I must be missing an obvious parameter but I am into this particular issue for hours and can't get it to stick over zooms, even though basic markers on the same map do indeed stay on their lat/lng coordinates.

**Comment by Falke-Design** [comment]
Make sure that you have set the iconAnchor correct.
https://plnkr.co/edit/AD2KSFrmSEMfdfKk

**Reply by cgombola** [reply]
How are the icon settings calculated?

Edit: I would still like to know the above but I figured out my issue, the custom icons are transparent-background arrows and so the anchor point stays on my mark but the arrow appears to "move" when zooming, simply because there is nothing in the transparent anchor point. So, of course, the further you zoom out, the more the arrow appears to "move".

Thanks for your help, it got my head in the right direction!

**Reply by Falke-Design** [reply]
> How are the icon settings calculated?

Based of the size of your image. But the easiest way is to use a default marker and then adjust with try and error the custom marker. I updated the demo
